src/Flexkids/Provider.php

```diff
@@ -126,8 +126,10 @@ public static function additionalConfigKeys()
     }
 
     /**
-     * @param $idToken
-     * @return Provider
+     * Set the token.
+     *
+     * @param  string  $idToken
+     * @return $this
      */
     private function setIdToken($idToken)
     {
```

Comment:
Could we just add typres and remove the docblock?

Reply:
(same with others)

Reply:
cf. https://github.com/SocialiteProviders/Providers/pull/1079/commits/246cf0fbe08db05ff334c06202136d14081dd03a